src/core/client/admin/routes/Moderate/ModerateContainer.tsx

```diff
@@ -24,18 +25,43 @@ class ModerateContainer extends React.Component<Props> {
 
   public render() {
     const allStories = !this.props.match.params.storyID;
+    // TODO: (tessalt) get active route in a better way
+    const queueName = [
```

Comment:
this feels sloppy but I need to know the name of the active route so I can pass it down to the `SiteSelector` component which needs to be able to construct a URL